src/Model/Item/Bookkeeping.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Billogram\Model\Item;
+
+use Billogram\Model\CreatableFromArray;
+
+/**
+ * @author Ibrahim Hizeoui <ibrahimhizeoui@gmail.com>
+ */
+class Bookkeeping implements CreatableFromArray
+{
+    /**
+     * @var string
+     */
+    private $incomeAccount;
+
+    /**
+     * @var string
+     */
+    private $vatAccount;
+
+    public function __construct()
```

Comment:
We do not need empty constructor. Either add some logic here or remove constructors